packages/mongo-nav/README.md

```diff
@@ -39,7 +39,7 @@ npm install @leafygreen-ui/mongo-nav @leafygreen-ui/leafygreen-provider@1.1.0
 
 | Prop                       | Type                                                                            | Description                                                                                                                                                                                                                                                                                                                              | Default                                                   |
 | -------------------------- | ------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------- |
-| `activeProduct`            | `'account', 'charts', 'cloud', 'realm', 'support', 'university'`                | Describes what product is currently active                                                                                                                                                                                                                                                                                               |                                                           |
+| `activeProduct`            | `Product`                                                                       | Describes what product is currently active                                                                                                                                                                                                                                                                                               |                                                           |
```

Comment:
Doesn't this mean we need to define that type at the bottom?

Reply:
👍